website/server/models/group.js

```diff
@@ -520,6 +523,9 @@ schema.methods.startQuest = async function startQuest (user) {
         });
     });
   });
+  this.sendChat(`Your quest, ${quest.text('en')}, has started.`, null, {
```

Comment:
But this is going in to party chat, not to a specific user, and we don't know what language the majority of the party speak. If the person who starts the quest speaks a language that none of the others do, then the translation won't be helpful (assuming it's that person's language that's used). Until we have a solution for party system messages to be translated into whatever language the reader is using, they should stay as English.